assets/common/sidecars/kube_rbac_proxy.yaml

```diff
@@ -0,0 +1,27 @@
+kind: Deployment
+apiVersion: apps/v1
+spec:
+  template:
+    spec:
+      containers:
+        - name: kube-rbac-proxy-${LOCAL_METRICS_PORT}
+          args:
+          - --secure-listen-address=0.0.0.0:${EXPOSED_METRICS_PORT}
+          - --upstream=http://127.0.0.1:${LOCAL_METRICS_PORT}/
+          - --tls-cert-file=/etc/tls/private/tls.crt
+          - --tls-private-key-file=/etc/tls/private/tls.key
+          - --tls-cipher-suites=${TLS_CIPHER_SUITES}
+          - --logtostderr=true
+          image: ${KUBE_RBAC_PROXY_IMAGE}
+          imagePullPolicy: IfNotPresent
+          ports:
+          - containerPort: ${EXPOSED_METRICS_PORT}
+            name: ${PORT_NAME}
+            protocol: TCP
+          resources:
+            requests:
+              memory: 20Mi
+              cpu: 10m
+          volumeMounts:
+          - mountPath: /etc/tls/private
+            name: metrics-serving-cert
```

Comment:
So this particular kube-rbac-proxy is for CSI driver itself. For other containers the rbac proxy sidecar is named accordingly but this one sticks out a bit. May be we should change it?

Reply:
Renamed to driver_kube_rbac_proxy.yaml + added a comment to the yaml file.